tests/wpt/metadata-layout-2020/css/css-ui/outline-with-padding-001.html.ini

```diff
@@ -0,0 +1,2 @@
+[outline-with-padding-001.html]
+  expected: FAIL
```

Comment:
Due to duplicated from metadata, this might be an intermittent FAIL while syncing 🤔 ? (It didn't fail in Linux builds in this r+ though)

```
  ▶ PASS [expected FAIL] /css/css-ui/outline-with-padding-001.html
```